examples/utils/utils.h

```diff
@@ -16,5 +19,10 @@ std::string read_file(const std::string& module_path) {
   return buffer.str();
 }
 
+std::shared_ptr<grpc::ChannelCredentials> get_credentials(bool insecure) {
+  return insecure ? grpc::InsecureChannelCredentials()
```

Comment:
As discussed, I think this should always return asylo credentials, but perhaps what's inside that can be parametrized by the `insecure` flag (e.g. null credentials vs SGX remote attestation verifier).